### ISSUE TITLE
Poisoning scenario - separate models for training on poisoned and cleaned datasets

### ISSUE THREAD

**Issue by davidslater** [twosixlabs/armory#582]
Separate training configurations for training on the poisoned dataset versus the cleaned dataset (number of epochs, learning rate, even models)

**Comment by davidslater** [comment]
MIT Request.
